flights/berks_ceh_insurances_flight.yaml

```diff
@@ -30,8 +30,8 @@ entityDefinitions:
         - !<transforms.ConcatTransform>
           columns: ["Medicaid", "NoMedicaidReason"]
           separator: "_"
-      general.status:
-        type: "general.status"
+      ol.statusdetails:
```

Comment:
This needs to be re-changed back to general.status. It was fine - what we wanted to change was everything that is ol.notes, to instead go into ol.statusdetails. So please change that property below.

Reply:
in new 317ce78